pom.xml

```diff
@@ -162,6 +169,7 @@
                         <param>edu.ucsb.cs156.happiercows.controllers.FrontendController</param>
                         <param>edu.ucsb.cs156.happiercows.controllers.FrontendProxyController</param>
                         <param>edu.ucsb.cs156.happiercows.services.CurrentUserServiceImpl</param>
+                        <param>edu.ucsb.cs156.happiercows.ExampleApplication</param>
```

Comment:
We have "HappierCowsApplication" which belongs to happy cows, but "ExampleApplication" is part of the courses-search app.

We could possibly remove this

Reply:
Good catch; I'll fix this.   It should say `HappierCowsApplication`.

We do need this; otherwise there will likely be a test coverage gap.

Reply:
Ah, I see that you are correct; we already have a line for `HappierCowsApplication`, so I just removed the `ExampleApplication` line.